### PR TITLE
RatioMaster: Add version 0.43

### LINKED ISSUE #9129: [Request] RatioMaster
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [X] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [X] the latest stable version of the program

### Information

**Name:** RatioMaster

**Description:** Standalone application which fakes upload and download stats of a torrent

**Homepage:** https://ratiomaster.net/

**Download link(s):** https://github.com/NikolayIT/RatioMaster.NET/releases/download/0.43/RatioMaster.NET_0.43.zip

**Some indication of popularity/repute:** 254 ⭐and 57 forks

### PR DESCRIPTION
**Pull request by AbbalYouness**
Closes #9129

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ratiomaster.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "license": "MIT",
+    "url": "https://github.com/NikolayIT/RatioMaster.NET/releases/download/0.43/RatioMaster.NET_0.43.zip",
+    "hash": "e3f415a2d9c7e9eccb926c85d78bb3acd3840990f25a946a894d00b0db7f6ba2",
+    "shortcuts": [
+        [
+            "RatioMaster.NET.exe",
+            "RatioMaster.NET"
+        ]
+    ],
+    "checkver": "github",
```
Comment (issaclin32) [inline bucket/ratiomaster.json:14]:
```suggestion
    "checkver": {
        "github": "https://github.com/NikolayIT/RatioMaster.NET"
    },
```
`"checkver": "github"` is only applicable if **"homepage"** is a Github repo

File: bucket/ratiomaster.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "autoupdate": {
+        "url": "https://github.com/NikolayIT/RatioMaster.NET/releases/download/$version/*$version.zip"
```
Comment (issaclin32) [inline bucket/ratiomaster.json:16]:
```suggestion
        "url": "https://github.com/NikolayIT/RatioMaster.NET/releases/download/$version/RatioMaster.NET_$version.zip"
```
Wildcard (`*`) is not supported for url

**Comment by AbbalYouness** [comment]
Thanks for the review !

**Pull request merged**